### PR TITLE
Add support for custom route prefix and custom model routeKeyName

### PR DESCRIPTION
**Pull request by Geelik**
Hi,

Love your package but i think it missed these features.
If you have problems with the way i implemented this, tell me and i will correct it.

Have a nice day / night

### CODE REVIEW

**Comment by zanechua** [comment]
Thanks for the PR.

However could you resubmit your PR with files that you have actually made modifications to and not just files that have been changed due to padding/spacing or recompilation of assets unless you have made modifications to the asset files?

File: src/Http/Controllers/OtterViewController.php [hunk 1]
```diff
@@ -17,10 +17,12 @@ public function __construct(Request $request)
             $this->baseResourceName = Otter::getClassNameFromRouteName($this->resourceName);
             $this->resource = $this->resourceNamespace.$this->baseResourceName;
             $this->prettyResourceName = str_singular(ucwords(str_replace('_', ' ', $this->resourceName)));
+	        
+            if (class_exists($this->resource))
+                $this->resourceRouteKeyName = $this->resource::$routeKeyName;
             /* @var TYPE_NAME $model */
-            $this->modelName = ($request->is('otter')) ? null : $this->resource::$model;
+            $this->modelName = ($request->is(config('otter.uri', 'otter'))) ? null : $this->resource::$model;
```
Comment (zanechua) [inline src/Http/Controllers/OtterViewController.php]:
```suggestion
            $this->modelName = ($request->is(config('otter.path', 'otter'))) ? null : $this->resource::$model;
```

Let's use they word 'path' instead, following telescope's config values

**Comment by Geelik** [comment]
I've done the changes.

**Comment by zanechua** [comment]
You seem to be missing all the modifications required for the Vue Components for this to work properly.

Have you tested your changes?

**Comment by Geelik** [comment]
You're right i'll fix that when i'm home.

**Comment by Geelik** [comment]
I've added all files. And yes i tested it.

**Comment by zanechua** [comment]
I'm going to have to pull down your PR and make some changes to it. From what i see for most of Vue Components, you made only single line modifications but the paddings or spacings are being adjusted for the entire file that the diff for the file is showing the entire file as being changed. One exception to this is the Table Component. Any chance you can just make the change to the single line and recommit?

**Comment by Geelik** [comment]
It's strange when i commit i don't have diff in spacing. I don't know how to correct this myself sadly.

**Comment by zanechua** [comment]
No worries.

I'll amend it when I pull it down for testing.

**Comment by zanechua** [comment]
Resolved the issues. Will merge this into develop for now for further testing.

**Pull request merged**